src/core/unplugin-factory.ts

```diff
@@ -197,14 +208,49 @@ const unpluginFactory: UnpluginFactory<FaviconsIconsPluginOptions | FaviconsLogo
                 }
             });
         },
+        transform(code, id) {
+            const runtimePackageName = `${packageName}/runtime`;
+            if (!code.includes(runtimePackageName)) {
+                return undefined;
+            }
+
+            const s = new MagicString(code);
+
+            const statements = findStaticImports(code).filter((index) => index.specifier === runtimePackageName);
+            if (statements.length === 0) {
+                return undefined;
+            }
+
+            statements.forEach((index) => {
+                const im = parseStaticImport(index);
+                const html = parsedHtml.map((tag) => tag.fragment).join("");
+                const generatedCode = `const ${im.defaultImport ?? im.imports} = ${genString(html)};`;
+                s.overwrite(im.start, im.end, generatedCode);
+            });
+
+            if (!s.hasChanged()) {
+                return undefined;
+            }
+
+            return {
+                code: s.toString(),
+                map: s.generateMap({
+                    includeContent: true,
+                    source: id,
+                }),
+            };
+        },
```

Comment:
Based on this code: https://github.com/unjs/magic-regexp/blob/main/src/transform.ts